libs/langchain/tests/integration_tests/cache/test_redis_cache.py

```diff
@@ -11,6 +11,18 @@
 REDIS_TEST_URL = "redis://localhost:6379"
 
 
+def test_redis_cache_ttl() -> None:
+    import time
+
+    import redis
+
+    langchain.llm_cache = RedisCache(redis_=redis.Redis.from_url(REDIS_TEST_URL), ttl=1)
+    langchain.llm_cache.update("foo", "bar", [Generation(text="fizz")])
+    key = langchain.llm_cache._key("foo", "bar")
+    time.sleep(1.1)
```

Comment:
Let's not use sleep in unit tests -- it slows down tests when folks try to run them

Could we instead use 

```
https://redis.io/commands/ttl/
```
or
```
https://redis.io/commands/pttl/
```

To verify that the ttl associated with the key is more than 0?